cmd/sonobuoy/app/operations/run.go

```diff
@@ -20,7 +20,9 @@ import (
 	"bytes"
 	"io"
 
+	// TODO - remove kubeconfig too.
```

Comment:
We need a kubeconfig somewhere so we can actually execute. What do you intend to replace it withL